.github/workflows/ci.yml

```
@@ -42,34 +45,16 @@ jobs:
       - uses: codecov/codecov-action@v1
         with:
           file: lcov.info
+
   aqua:
     name: Aqua tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: julia-actions/setup-julia@v1
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
         with:
           version: '1'
-      - uses: julia-actions/cache@v1
+      - uses: julia-actions/cache@v2
       - uses: julia-actions/julia-buildpkg@v1
       - run: julia -e 'using Pkg; Pkg.add("Aqua")'
       - run: julia --project=@. -e 'using Aqua, DustExtinction; Aqua.test_all(DustExtinction, ambiguities=false)'
```

```suggestion
      - run: julia --color=yes -e 'using Pkg; Pkg.add("Aqua")'
      - run: julia --color=yes --project=@. -e 'using Aqua, DustExtinction; Aqua.test_all(DustExtinction, ambiguities=false)'
```
also same comment as https://github.com/JuliaAstro/BoxLeastSquares.jl/pull/3#discussion_r1679683803